lib/rubocop/cop/rails/transaction_exit_statement.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks for the use of exit statements (namely `return`,
+      # `break` and `throw`) in transactions.
+      #
+      # @example
+      #   # bad
+      #   ApplicationRecord.transaction do
+      #     return if user.active?
+      #     break if user.active?
+      #     throw if user.active?
+      #   end
```

Comment:
Perhaps it's better to separate the bad examples.

```suggestion
      #   # bad
      #   ApplicationRecord.transaction do
      #     return if user.active?
      #   end
      #
      #   # bad
      #   ApplicationRecord.transaction do
      #     break if user.active?
      #   end
      #
      #   # bad
      #   ApplicationRecord.transaction do
      #     throw if user.active?
      #   end
```